modules/apps/asset/asset-display-page-service/src/main/java/com/liferay/asset/display/page/internal/upgrade/v1_0_0/UpgradeAssetDisplayPage.java

```diff
@@ -14,20 +14,89 @@
 
 package com.liferay.asset.display.page.internal.upgrade.v1_0_0;
 
+import com.liferay.asset.display.page.constants.AssetDisplayPageConstants;
+import com.liferay.journal.model.JournalArticle;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
 import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.util.LoggingTimer;
 import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.uuid.PortalUUIDUtil;
+
+import java.sql.Date;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Timestamp;
 
 /**
  * @author Vendel Toreki
  */
 public class UpgradeAssetDisplayPage extends UpgradeProcess {
 
-	public UpgradeAssetDisplayPage() {
+	public UpgradeAssetDisplayPage(
+		ClassNameLocalService classNameLocalService) {
+
+		_classNameLocalService = classNameLocalService;
 	}
 
 	@Override
 	protected void doUpgrade() throws Exception {
 		upgradeSchema();
+
+		upgradeAssetDisplayPageEntry();
+	}
+
+	protected void upgradeAssetDisplayPageEntry() throws Exception {
```

Comment:
It makes sense to move this upgrade step to journal-service module since it is more about JournalArticle behaviour, not AssetDisplayPageEntry.